metadata_mapper/lambda_shepherd.py

```diff
@@ -55,24 +56,31 @@ def map_collection(payload, context):
     if len(missing_enrichments) > 0:
         print(f"[{collection_id}]: Missing enrichments: {missing_enrichments}")
 
+    count = 0
+    page_count = 0
     if settings.DATA_SRC == 'local':
         vernacular_path = settings.local_path(
             'vernacular_metadata', collection_id)
-        page_list = [f for f in os.listdir(vernacular_path)
-                     if os.path.isfile(os.path.join(vernacular_path, f))]
-        count = 0
+        try:
+            page_list = [f for f in os.listdir(vernacular_path)
+                         if os.path.isfile(os.path.join(vernacular_path, f))]
+        except FileNotFoundError as e:
+            logging.debug(f"{e} - have you fetched {collection_id}?")
+            return {
+                'statusCode': 400,
+                'body': json.dumps({
+                    'error': (
+                        f"{repr(e)} - have you fetched {collection_id}? ",
+                        f"looked in {e.filename}"
```

Comment:
I was confused about this line at first, but it's reporting the name of the directory we looked for files in, yes? Super minor, but maybe this would make it clearer when reading the code:

```
Looked in dir {e.filename}
```

Reply:
Done